paasta_tools/monitoring_tools.py

```diff
@@ -55,6 +55,8 @@ def get_replication_for_instance(
     yelp_meteorite = None
 
 
+DEFAULT_REPLICATION_RUNBOOK = "y/paastatools"
```

Comment:
I agree y/paastatools isn't the best page for this. I think Section 3 of y/stuckbounce covers the same steps you would use for under-replication. Maybe you could pull that out into a separate page and link to it from both y/stuckbounce and the alerts.

Reply:
Done. I created y/unhealthy-paasta-instances.